medical_patient_occupation/models/medical_patient_occupation.py

```diff
@@ -21,17 +21,15 @@
 #
 ###############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models
 
 
-class MedicalOccupation(orm.Model):
-    _name = 'medical.occupation'
+class MedicalPatientOccupation(models.Model):
+    _name = 'medical.patient.occupation'
+
+    code = fields.Char(size=256, )
+    name = fields.Char(size=256, required=True, translate=True, )
 
-    _columns = {
-        'code': fields.char(size=256, string='Code'),
-        'name': fields.char(size=256, string='Name', required=True,
-                            translate=True),
-    }
     _sql_constraints = [
         ('name_uniq', 'UNIQUE(name)', 'Name must be unique!'),
```

Comment:
Yup - from what I understand, there aren't really overlapping Ethnicity codes; but Occupations sometimes don't have codes, or overlap depending on how they're divided. Now that I'm looking though, I also meant to add a `required=True` to ethnicity codes.